tests/jax/models/MNIST/CNN/MNIST_CNN.py

```diff
@@ -0,0 +1,33 @@
+# SPDX-FileCopyrightText: (c) 2024 Tenstorrent AI ULC
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import jax
+from flax import linen as nn
+
+
+class MNIST_CNN(nn.Module):
+    @nn.compact
+    def __call__(self, x, *, train: bool):
+        x = nn.Conv(features=32, kernel_size=(3, 3), padding="SAME")(x)
+        x = nn.BatchNorm(use_running_average=not train, epsilon=1e-5)(x)
+        x = nn.relu(x)
+        x = nn.max_pool(x, window_shape=(2, 2), strides=(2, 2))
+
+        x = nn.Conv(features=64, kernel_size=(3, 3), padding="SAME")(x)
+        x = nn.BatchNorm(use_running_average=not train, epsilon=1e-5)(x)
+        x = nn.relu(x)
+        x = nn.max_pool(x, window_shape=(2, 2), strides=(2, 2))
+
+        x = x.reshape((x.shape[0], -1))
+        x = nn.Dense(features=256)(x)
+        x = nn.BatchNorm(use_running_average=not train, epsilon=1e-5)(x)
+        x = nn.relu(x)
+
+        x = nn.Dense(features=128)(x)
+        x = nn.BatchNorm(use_running_average=not train, epsilon=1e-5)(x)
+        x = nn.relu(x)
+        x = nn.Dense(features=10)(x)
+        # TODO(stefan): This causes a failure, uncomment once fixed.
```

Comment:
Is the failure something related to tt-xla or is it failure later in compilation/runtime? If it is related to tt-xla, please open a new GH issue and paste the link in this TODO comment. If it is issue in compilation/runtime then you don't have to comment this out, you can just update the skip reason in the tester.

Reply:
It's a segfault deep inside LLVM, possibly some fusion or decomposition gone awry. Initially I didn't want to keep the change as in that case we no longer have any tests that fail on broadcasting specifically, but ideally that too would be covered by another test. I'm changing this and bumping up tests with BNs and broadcasting on my todo list.